data/DOH/data.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "DOHCOIN",
+    "symbol": "DOH",
+    "decimals": 18,
+    "description": "DOHCOIN Token mined with DOH MINER NFTs.",
+    "website": "https://dohcoin.com",
+    "twitter": "@dohwtf",
+    "tokens": {
+      "base": {
+        "address": "0xF37923311D71c986D701cddb3394777322294E3a"
```

Comment:
I notice there is no ethereum address. Can you clarify if this is a native l2 token that doesn't have an l1 address? If so we need to fix our validation to handle this

Reply:
This is a native l2 base token only.

Reply:
thanks for clarifying!